scripts/jenkins/groovy/defineTestStages.groovy

```diff
@@ -529,23 +537,19 @@ def call(final pipelineContext) {
       timeoutValue: 90, component: pipelineContext.getBuildConfig().COMPONENT_PY
     ],
     [
-      stageName: 'Py3.10 Single Node', target: 'test-pyunit-single-node', pythonVersion: '3.10',
```

Comment:
These tests are also only for oldest and latest java?

Reply:
yes, oldest and latest supported python